### PR TITLE
SUPPORT.MD proposal

### PR DESCRIPTION
**Pull request by jeffwilcox**
The CSS (Microsoft Customer Service and Support) team has a process
for providing support to some products or projects. To help improve
the experience for such cases, this update proposes a new SUPPORT.MD
file to be placed in the default repo templates with information for
maintainers to update and publish.

For questions in the proposal, please reach out to JHornick or comment
on this original pull request.

This PR is originally targeting the `mit` (the standard Microsoft new project repo type).

### CODE REVIEW

File: projections/mit/SUPPORT.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+## How to file issues and get help  
+
+This project uses GitHub Issues to track bugs and feature requests. Please search the existing 
+issues before filing new issues to avoid duplicates.  For new issues, file your bug or 
+feature request as a new Issue.
+
+For help and questions about using this project, please **REPO MAINTAINER: INSERT INSTRUCTIONS HERE 
+FOR HOW TO ENGAGE REPO OWNERS OR COMMUNITY FOR HELP. COULD BE A STACK OVERFLOW TAG OR OTHER
+CHANNEL. WHERE WILL YOU HELP PEOPLE?].
```
Comment (JamieMagee) [inline projections/mit/SUPPORT.md:21]:
```suggestion
CHANNEL. WHERE WILL YOU HELP PEOPLE?].**

```

**Comment by johngossman** [comment]
lgtm

File: projections/mit/SUPPORT.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+## Microsoft Support Policy  
+
+Support for this **PROJECT or PROJECT** is limited to the resources listed above.
```
Comment (andschwa) [inline projections/mit/SUPPORT.md:25]: What's this mean "**PROJECT or PROJECT**"?

File: projections/mit/SUPPORT.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+- **Yes CSS support:** Fill out an intake form at [aka.ms/spot](https://aka.ms/spot). CSS will work with/help you to determine next steps. More details also available at [aka.ms/onboardsupport](https://aka.ms/onboardsupport).
+- **Not sure?** Fill out a SPOT intake as though the answer were "Yes". CSS will help you decide.
+
+*Then remove this first heading from this SUPPORT.MD file before publishing your repo.*
+
+# Support
+
+## How to file issues and get help  
+
+This project uses GitHub Issues to track bugs and feature requests. Please search the existing
```
Comment (thomastaylor312) [inline projections/mit/SUPPORT.md:15]: Can we auto generate a link to the issues page of the new repo? Basically make it so "Github Issues" and "Issue" (down below) are hyperlinked to the issues page

Comment (jeffwilcox) [reply projections/mit/SUPPORT.md:15]: Wish so, but it's not currently a feature/capability of the system.

**Review by thomastaylor312 — APPROVED** [review]

File: projections/mit/SUPPORT.md (hunk 2 above)
Comment (jeffwilcox) [reply projections/mit/SUPPORT.md:25]: I think the guidance from @jhornick-ms is that they want the maintainer to clarify if it's a product or a project... products might be something like Cosmos DB vs a project like "some Cosmos integration for my product", maybe?

I'll let @jhornick-ms comment

File: projections/mit/SUPPORT.md (hunk 1 above)
Comment (jeffwilcox) [reply projections/mit/SUPPORT.md]: fixed

File: projections/mit/SUPPORT.md (hunk 2 above)
Comment (andschwa) [reply projections/mit/SUPPORT.md:25]: I wondered, in which case this should probably be "PROJECT or PRODUCT" (unless my eyes are deceiving me, it says "project" twice).

Comment (jhornick-ms) [reply projections/mit/SUPPORT.md:25]: @andschwa - @jeffwilcox  is right - this verbiage was intended to allow the repo owners to indicate whether this is a project, or something that rises to the level of "product" which is more likely - but not required - to carry official support.  There are entities which differentiate between the two, so we started with this in an attempt to embrace those terms.  If it's confusing, we could alter to "Support for the contents of this repo is limited to the resources listed above."

Comment (bureado) [reply projections/mit/SUPPORT.md:25]: It's a typo.

**Pull request merged**